tests/addon/test_unit.py

```diff
@@ -2906,3 +2908,69 @@ def test_with_migrating_content_from_old_note_type(self, use_new_templates: bool
             template["afmt"],
             options=NamerFactory.with_parameters("afmt", use_new_templates),
         )
+
+
+def test_get_daily_review_data_since_last_sync(mocker, anki_session_with_addon_data):
+    with anki_session_with_addon_data.profile_loaded():
+        last_sync = datetime.now() - timedelta(days=2)
+        yesterday = datetime.now() - timedelta(days=1)
+        mock_rows = [
+            (int(datetime.timestamp(last_sync) * 1000) + 1000, 1, 30),
+            (int(datetime.timestamp(last_sync) * 1000) + 2000, 2, 40),
+            (int(datetime.timestamp(yesterday) * 1000) - 1000, 3, 50),
+            (int(datetime.timestamp(yesterday) * 1000) - 500, 4, 60),
+        ]
+
+        mocker.patch(
+            "ankihub.main.review_data.aqt.mw.col.db.all", return_value=mock_rows
+        )
+        mock_ms_timestamp_to_datetime = mocker.patch(
+            "ankihub.main.review_data._ms_timestamp_to_datetime"
+        )
+        mock_ms_timestamp_to_datetime.side_effect = lambda ts: datetime.fromtimestamp(
+            ts / 1000
+        )
+
+        result = get_daily_review_data_since_last_sync(last_sync)
+
+        expected_data = [
+            DailyCardReviewSummaryData(
+                total_cards_studied=2,
+                total_time_reviewing=70,
+                total_cards_marked_as_again=1,
+                total_cards_marked_as_hard=1,
+                total_cards_marked_as_good=0,
+                total_cards_marked_as_easy=0,
+                review_session_date=(last_sync + timedelta(seconds=1)).date(),
+            ),
+            DailyCardReviewSummaryData(
+                total_cards_studied=2,
+                total_time_reviewing=110,
+                total_cards_marked_as_again=0,
+                total_cards_marked_as_hard=0,
+                total_cards_marked_as_good=1,
+                total_cards_marked_as_easy=1,
+                review_session_date=(yesterday - timedelta(seconds=1)).date(),
```

Comment:
:pick:It seems like we don't need to subtract the 1 second here?  (Same as the last comment.)